internal/pkg/task/config_runner.go

```diff
@@ -104,3 +106,12 @@ func (r *ConfigRunner) validateDependencies() error {
 
 	return nil
 }
+
+// CheckNonZeroExitCode returns the status of the containers part of the given tasks.
+func (r *ConfigRunner) CheckNonZeroExitCode(tasks []*Task) error {
```

Comment:
Leaving the comment here just for future reference, no action item for ya!

At a latter point we should refactor this package such that:
1. Instead of returning tasks, `Run()` should return cache the tasks.
2. This function will no longer need to have `tasks` as the parameter. It should use the cache `tasks` instead.
3. Same for `cluster` for `EnvRunner`. We should cache it, instead of getting cluster twice, once in `Run()`, once in `CheckExitCode`.

The above mentioned refactoring is related to: https://github.com/aws/copilot-cli/issues/3469#issuecomment-1124393633